packages/web-components/src/components/rux-container/rux-container.scss

```diff
@@ -87,6 +87,9 @@
     background: var(--color-background-surface-header);
     padding: 0 var(--spacing-4) calc(var(--spacing-1) + 2px) var(--spacing-4); //0 16px 6px 16px
 }
+.rux-container__tab-bar-compact {
```

Comment:
Deals with the extra padding on tab-bar div. We don't want that with this new compact style.